Update parent idling info (#6200)
ClickHouse/clickhouse-docs: docs/cloud/features/05_infrastructure/warehouses.md

- **There is currently a soft limit of 5 services per warehouse.** Contact the support team if you need more than 5 services in a single warehouse.
- **One replia Primary Sservice** Today, the default behavior is the secondary services can have one replica, the primary service must have at least 2.
  To enable single replica primary services, please contact support. This behavior will be enabled by default in Q2 2026.

> - **Primary service idling** Previously, the default behavior is that the primary service cannot auto-idling. As of May 2026, Parent service auto-idling is enabled by default. Existing services will have access to the feature, new services will have it enabled by default. 

Author: ```suggestion
- **Primary service idling** Previously, the default behavior is that the primary service cannot auto-idling. As of May 2026, primary service auto-idling is enabled by default. As part of the rollout, existing services have access to the feature while new services created post have it enabled by default. 
```

> - **Primary service idling** Previously, the default behavior is that the primary service cannot auto-idling. As of May 2026, primary service auto-idling is enabled by default. As part of the rollout, existing services have access to the feature while new services created post have it enabled by default. 

Reviewer: ```suggestion
- **Primary service idling**: Previously, primary services could not auto-idle by default. As of May 2026, primary service auto-idling is enabled by default. As part of the rollout, existing services have access to the feature while new services created post rollout have it enabled by default. 
```
